patches/protobuf.patch

```diff
@@ -1,28 +1,30 @@
 diff --git a/CMakeLists.txt b/CMakeLists.txt
 new file mode 100644
-index 0000000..f74088f
+index 000000000..d73089c43
```

Comment:
I'm not really sure what to do with this, as my git was a little more updated....

Reply:
this is a common thing as old patch files get updated